tunnel/config.go

```diff
@@ -148,6 +141,26 @@ func (r SSHConfigFile) getKey(host string) string {
 	return ""
 }
 
+// SSHConfigFilePath is a path to the ssh config file
+type SSHConfigFilePath string
+
+// NewSSHConfigFilePath creates a new instance of SSHConfigFilePath
+// from customPath or fallback to $HOME/.ssh/config
+func NewSSHConfigFilePath(customPath string) (SSHConfigFilePath, error) {
```

Comment:
How about merging this function to `NewSSHConfigFile`?
Once the config flag has a default value, it can not accept the given `customPath` variable with empty vaue.